sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -455,6 +456,8 @@ def test_partition():
     assert partition(p).is_positive
     assert partition(x).subs(x, 7) == 15
     assert partition(y).subs(y, 8) == 22
+    assert [nT(i, j) for i in range(5) for j in range(i + 2)] == [
```

Comment:
Is this valid also for `_nT`?

Reply:
Yes, nT Function calls _nT when i and j is integer.

Reply:
I seem to get a different result:
```
In [2]: [_nT(i, j) for i in range(5) for j in range(i + 2)]
Out[2]: [0, 0, 0, 1, 0, 0, 1, 1, 0, 0, 1, 1, 1, 0, 0, 1, 2, 1, 1, 0]
```
Perhaps this should test the private `_nT` instead of `nT`.

Reply:
@jksuom  I have changed he code little bit. Now it should give correct result. Please review this.